prow/cmd/deck/static/pr/pr.ts

```diff
@@ -1132,6 +1136,13 @@ function closestMatchingQueries(pr: PullRequest, queries: TideQuery[]): Processe
     return processedQueries;
 }
 
+/**
+ * Normalizes GitHub login strings
+ */
+function normLogin(login: string = ""): string {
+    return login.toLowerCase().replace(/^@/, "");
```

Comment:
Is this replacement necessary? I think it would make more sense for this to match the behavior of the go version of `NormLogin`.

Reply:
The `replace` here is the JS equivalent of the Go `TrimPrefix`.
https://github.com/kubernetes/test-infra/blob/5c17574a9a87e3fce30d20925bf8ed50ed533f64/prow/github/types.go#L175-L178

Reply:
Oh I didn't realize we handled the `@` prefix.